network/network_client.ml

```diff
@@ -28,16 +28,16 @@ let retry_econnrefused f =
     | None -> loop () in
   loop ()
 
-module Client = Network_interface.Client(struct
-  let rpc call =
-    retry_econnrefused
-      (fun () ->
-	if !use_switch
-	then json_switch_rpc !queue_name call
-        else xml_http_rpc
-          ~srcstr:(Xcp_client.get_user_agent ())
-          ~dststr:"network"
-          Network_interface.uri
-	  call
-      )
-end)
+let rpc call =
```

Comment:
In the other modules we are moving away from global openings. Please remove `open Network_interface` and `open Xcp_client`, see e.g. https://github.com/xapi-project/xcp-idl/blob/master/gpumon/gpumon_client.ml

Reply:
Done